src/frontend/App.js

```diff
@@ -1,41 +1,16 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Home, ProductListing } from "./Pages"
+import { Routes, Route } from 'react-router-dom'
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Routes>
+        <Route path='/' element={<Home />} />
+        <Route path="/products" element={<ProductListing />} />
+
+      </Routes>
```

Comment:
Suggestion: You can make one route component where all the routes will be placed so your app component looks cleaner.

Reply:
I have made a seperate component for Routes thanks for the suggestion